region/crateria/west/Gauntlet Entrance.json

```diff
@@ -228,24 +228,63 @@
       "link": [1, 2],
       "name": "Base",
       "requires": [
-        "ScrewAttack"
+        "ScrewAttack",
+        {"or": [
+          "canCarefulJump",
+          {"acidFrames": 15}
+        ]}
       ],
       "clearsObstacles": ["A"]
     },
     {
       "link": [1, 2],
       "name": "Gauntlet Entrance with Bombs (Left to Right)",
       "requires": [
-        "h_canUseMorphBombs"
+        "h_canUseMorphBombs",
+        {"or": [
+          "canTrickyJump",
+          {"ammo": {"type": "Super", "count": 2}},
+          {"acidFrames": 100}
+        ]},
+        {"or": [
+          "canDodgeWhileShooting",
+          "Wave",
+          {"enemyDamage": {
+            "enemy": "Waver",
+            "type": "contact",
+            "hits": 2
+          }}
+        ]},
+        {"or": [
+          "canCarefulJump",
```

Comment:
Spring helps, but might not be worth including. Would it be clear to a player without careful jump know not to break the middle bomb wall completely, in order to stay out of the acid?

Reply:
Springball at Basic would help with every other Bomb wall to keep toes out of the acid. 
Springball would be more clear than this expected jump morph 🤔 Bomb gauntlet doesn't sound fun; both suits roll on the ground would be the easiest thing to do, but then you are suitless lava diving.  And eating a lot of damage that a jump morph would fix.  Space jump allows that to be possible.